darts/tests/test_timeseries_plot.py

```diff
@@ -37,40 +37,37 @@ class TestTimeSeriesPlot:
     @patch("matplotlib.pyplot.show")
     @pytest.mark.parametrize(
         "config",
-        product(
-            ["dt", "ri"],
-            ["d", "p"],
-            [True, False],
-        ),
+        product(["dt", "ri"], ["d", "p"], [True, False], [True, False]),
```

Comment:
previous formatting was a bit more readable imo :)

```suggestion
        product(
            ["dt", "ri"],
            ["d", "p"],
            [True, False],
            [True, False],
        ),
```